query-helpers/profile.js

```diff
@@ -44,7 +44,9 @@ const profileSQL = (profile, ids, isPrevious = false) => `
   /*
    * an aggregation of enriched selection, joined with base that aggregates
    * e and m for all rows for a given 'variable' in the selection, and adds
-   * additional aggregate values cv, percent, percent_m, and is_reliable
+   * additional aggregate values cv, percent, percent_m, and is_reliable.
+   * Note: m is coalesced to 0 if the value does not exist in the data
+   * TODO make this fix in the data? make m NOT NULL DEFAULT 0?
```

Comment:
I like this! I think this will be easier once we adopt more of the ETL process from population.